tests/sample.test.js

```diff
@@ -0,0 +1,7 @@
+const expect = require('chai').expect;
+
```

Comment:
Line 1 does not look like `ES 6`, please refactor your code to use something like 
`import { expect } from 'chai';`

Reply:
Refactored code to ES6